server.js

```diff
@@ -1,35 +1,110 @@
-import express from "express";
-import cors from "cors";
-import mongoose from "mongoose";
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
-mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
-mongoose.Promise = Promise;
+import express from 'express'
+import cors from 'cors'
+import mongoose from 'mongoose'
+import dotenv from 'dotenv'
```

Comment:
Also new for me, I think! Don´t really understand after googling, might be tired, haha:) Maybe you can tell me more about  it?

Reply:
It's for storing environment variables that can store secrets, Damien talks about it in the material. I just tried it out but it wasn't needed for this project in particular. The environment variables is stored in another folder in the project - so they can be reached from all over, the ones i tried stored secrets and are not pushed to github.